### PR TITLE
fixed the errors about empty arrays

### PR DESCRIPTION
**Pull request by shinobu**

### CODE REVIEW

File: application/tests/CodeSniffer/Standards/Ontowiki/Sniffs/Commenting/FileCommentSniff.php [hunk 1]
```diff
@@ -77,14 +78,18 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
             $git_year1 .='-';
             $git_year1 .=$git_year2;
           }
-          //$year = " * @copyright Copyright (c) " . date('Y') . ", {@link http://aksw.org AKSW}\n";
           $year = " * @copyright Copyright (c) " . $git_year1 . ", {@link http://aksw.org AKSW}\n";
           $this->copyright[4]= $year;
-        }
-        else {
+        } else {
+          echo count($tokens);
```
Comment (white-gecko) [inline application/tests/CodeSniffer/Standards/Ontowiki/Sniffs/Commenting/FileCommentSniff.php]: Should this `echo` really be here? Pleas review your changes for left over lines before committing or at least before sending the pull-request.

Comment (shinobu) [inline application/tests/CodeSniffer/Standards/Ontowiki/Sniffs/Commenting/FileCommentSniff.php]: sorry, that one slipped past my check

**Comment by white-gecko** [comment]
I still get `Notice: Undefined offset: 0` and `Notice: Undefined offset: 2` (altering):

```
Notice: Undefined offset: 0 in …/ontowiki/application/tests/CodeSniffer/Standards/Ontowiki/Sniffs/Commenting/FileCommentSniff.php on line 75

Call Stack:
    0.0001     229880   1. {main}() …/ontowiki/vendor/squizlabs/php_codesniffer/scripts/phpcs:0
    0.0110    2104552   2. PHP_CodeSniffer_CLI->runphpcs() …/ontowiki/vendor/squizlabs/php_codesniffer/scripts/phpcs:25
    0.0113    2162904   3. PHP_CodeSniffer_CLI->process() …/ontowiki/vendor/squizlabs/php_codesniffer/CodeSniffer/CLI.php:106
    0.0355    4469536   4. PHP_CodeSniffer->processFiles() …/ontowiki/vendor/squizlabs/php_codesniffer/CodeSniffer/CLI.php:914
   13.3738   12474272   5. PHP_CodeSniffer->processFile() …/ontowiki/vendor/squizlabs/php_codesniffer/CodeSniffer.php:629
   13.3739   12474536   6. PHP_CodeSniffer->_processFile() …/ontowiki/vendor/squizlabs/php_codesniffer/CodeSniffer.php:1739
   13.3739   12475240   7. PHP_CodeSniffer_File->start() …/ontowiki/vendor/squizlabs/php_codesniffer/CodeSniffer.php:1861
   13.3842   13273496   8. Ontowiki_Sniffs_Commenting_FileCommentSniff->process() …/ontowiki/vendor/squizlabs/php_codesniffer/CodeSniffer/File.php:573
```

**Comment by shinobu** [comment]
that's odd, i can't reproduce this error on my pc, did you used the fix/FileCommentSniff branch for testing? neither the git nor the non-git run throw errors for me

**Comment by white-gecko** [comment]
Yes I'm using the feature branch. Did you enable all PHP error outputs, also for PHP cli?

Natanael

Am 12. Mai 2016 20:06:47 MESZ, schrieb shinobu <notifications@github.com>:
>that's odd, i can't reproduce this error on my pc, did you used the
>fix/FileCommentSniff branch for testing? neither the git nor the
>non-git run throw errors for me
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/AKSW/OntoWiki/pull/352#issuecomment-218838539

-- 
Diese Nachricht wurde von meinem Mobiltelefon mit Linux Kernel und JVM Userland von K-9 Mail gesendet.

**Comment by shinobu** [comment]
i'm pretty sure it's enabled (i did saw those errors prior to fixing them)

**Comment by white-gecko** [comment]
Yes I'm on this commit, and get the notices d229668f64d9af29c852d560fcaedf6e113ea847

**Comment by white-gecko** [comment]
Notice: Undefined offset: ? in …/ontowiki/application/tests/CodeSniffer/Standards/Ontowiki/Sniffs/Commenting/FileCommentSniff.php on line 71|72|74|75

**Comment by white-gecko** [comment]
Try to reproduce it by checking out an extension into the extensions folder e.g. https://github.com/AKSW/csvimport.ontowiki

**Comment by shinobu** [comment]
fixed the error in 24aeae7

**Pull request merged**